src/protocols/arinc818.stanza

```diff
@@ -0,0 +1,160 @@
+doc: \<DOC>
+@brief ARINC-818 Protocol
+
+ARINC-818 is a serial high speed protocol supporting video links in avionics applications.
+Also known as Avionics Digital Video Bus over Fibre Channel.
+@see https://en.wikipedia.org/wiki/ARINC_818
+
+This functions and definitions in this file support defining ARINC818
+connections between sources and receivers on a printed circuit board.
+@see https://docs.wixstatic.com/ugd/aefb08_37d298772ed9485f97af8f2954270e38.pdf?index=true
+<DOC>
+#use-added-syntax(jitx)
+defpackage jsl/protocols/arinc818:
+  import core
+  import jitx
+  import jitx/commands
+
+  import jsl/errors
+  import jsl/bundles
+  import jsl/pin-assignment
+  import jsl/si/helpers
+  import jsl/si/pairs
+  import jsl/si/constraints
+
+doc: \<DOC>
+@brief ARINC Type
+This enumerated type defines either a TX type or RX type for the ARINC818 bundle.
+<DOC>
+public pcb-enum jsl/protocols/arinc818/ARINC818Type :
+  TX
+  RX
+
+doc: \<DOC>
+@brief ARINC Generation enums
+This is a fixed list of possible generation definitions for ARINC818
+<DOC>
+
+public pcb-enum jsl/protocols/arinc818/ARINC818Version:
+  ARINC818-FC1x   ;   1.0625 Gbits/s
+  ARINC818-1p5    ;   1.5    Gbits/s
+  ARINC818-1p62   ;   1.62   Gbits/s
+  ARNIC818-FC2x   ;   2.125  Gbits/s
+  ARINC818-2p5    ;   2.5    Gbits/s
+  ARINC818-3p1875 ;   3.1875 Gbits/s
+  ARNIC818-FC4x   ;   4.5    Gbits/s
+  ARINC818-5p0    ;   5.0    Gbits/s
+  ARINC818-FC6x   ;   6.375  Gbits/s
+  ARINC818-FC8x   ;   8.5    Gbits/s
+  ARINC818-FC12x  ;  12.75   Gbits/s
+  ARINC818-FC16x  ;  14.025  Gbits/s
+  ARINC818-FC24x  ;  21.0375 Gbits/s
+  ARINC818-FC32x  ;  28.05   Gbits/s
+
+
+
+doc: \<DOC>
+@brief ARINC818 Bundle
+The Data Bundle consists of 1 differential pair (1 RX or TX)
+@member TX/RX for the ARINC818 bundle (consists of either a TX or RX diff pair)
+<DOC>
+
+public pcb-bundle ARINC818 (type:ARINC818Type) :
+  name = "ARINC818"
+  description = "ARINC 818 AVDB protocol"
+  switch(type) :
+    TX : make-port(`TX, diff-pair)
+    RX : make-port(`RX, diff-pair)
```

Comment:
For other unidirectional standards like DisplayPort, we don't differentiate between transmit and receive. 

It feels like this attempting to implement a Cardinality style implementation. Ie - this is a output, this is an input, kind of thing.  

Side note - is the ARINC818 really just a diff-pair ? Do we have a copy of the spec ?